orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/CreateServerGroupStage.groovy

```diff
@@ -58,4 +68,67 @@ class CreateServerGroupStage extends AbstractDeployStrategyStage {
 
     return tasks
   }
+
+  @Override
+  List<Stage> onFailureStages(@Nonnull Stage stage) {
+    def stageData = stage.mapTo(StageData)
+
+    if (!stageData.rollbackOnFailure) {
```

Comment:
Is `rollbackOnFailure` a reasonable enough attribute ... or is there value in something along the lines of:

```
rollback: {
  onFailure: true
}
```

The latter might be easier to extend.

Reply:
Updated to use

```
rollback: {
  onFailure: true
}
```

vs

```
rollbackOnFailure: true
```